src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -136,6 +143,14 @@ public void create(final EventTypeBase eventType) throws TopicCreationException,
             }
             throw e;
         }
+        try {
+            nakadiKpiPublisher.publish(etLogEventType, () -> new JSONObject()
+                    .put("event_type", eventType.getName())
+                    .put("status", "created")
+                    .put("category", eventType.getCategory().toString()));
+        } catch (final Exception e) {
```

Comment:
It throws no exceptions, why did you wrap it in catch ?

Reply:
Just in case some unchecked exception is thrown. My point here was that if `nakadiKpiPublisher.publish` fails for whatever reason - that should not fail the actual request to Nakadi (creating/updating/deleting ET)

Reply:
it was designed not to fail during putting event into the queue, but if you are not certain, than I suggest to catch exceptions inside publish method, it will be cleaner.

Reply:
Ok, I moved the catch inside publish method https://github.com/zalando/nakadi/pull/799/commits/b7ebac82d12a99880f07c753b412fdfa6e42edb3